README.md

```diff
@@ -98,8 +109,8 @@ $("button").on("click", function(event){
 ```
 
 Your Answer:
-```js
-// write code here
+```
+Not sure.
 ```
```

Comment:
The equivalent would be using `.querySelector,` which returns the first DOM element that matches the selector you pass in as an argument.

```js
document.querySelector("button").addEventListener("click", function(event){
  event.target.style("color", "red");
};
```

You could also use `querySelectorAll`, which would return all of the buttons on the page. In that case, however, you would need to loop through the resulting array an add the event listener individually.